static/js/matomo.js

```diff
@@ -0,0 +1,16 @@
+var _paq = (window._paq = window._paq || []);
+/* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+_paq.push(['requireCookieConsent']);
+_paq.push(['trackPageView']);
+_paq.push(['enableLinkTracking']);
+(function () {
+  var u = 'https://matomo.iota-community.org/';
+  _paq.push(['setTrackerUrl', u + 'matomo.php']);
+  _paq.push(['setSiteId', '13']);
```

Comment:
Can you change the id to 14 for testing?
```suggestion
  _paq.push(['setSiteId', '14']);
```